### DISCUSSION TITLE
How to use poetry to maintain a local python install?

### DISCUSSION THREAD

**Discussion by justrajdeep** [python-poetry/poetry#9171] [Q&A]
Hi Experts

I am using poetry to maintain my own version of python since i cannot install a global python without admin access.

So i created a wrapper script like this ...

```bash
> \cat ~/bin/python               
#!/bin/bash

set -e

source ~/bin/scripts/emacs_python/.venv/bin/activate

~/bin/scripts/emacs_python/.venv/bin/python3 $@
```

it works for most parts but 

when i try to do 

```python
python -c "for i in range(1000): print(i)" 
```
it Errors

```
 python -c "for i in range(1000): print(i)"       
  File "<string>", line 1
    for
       ^
SyntaxError: invalid syntax
```

I can solve this if i symbolically link `~/bin/scripts/emacs_python/.venv/bin/python3` to `~/bin/python`  but then when i run my script it errors saying that my poetry added modules are not available.

Can someone please help?

**Comment by Secrus** [comment]
Poetry is not a python manager, it only uses the Pythons you have installed.
